packages/web-pkg/src/composables/sse/useServerSentEvents.ts

```diff
@@ -44,9 +46,17 @@ export const useServerSentEvents = (options: ServerSentEventsOptions) => {
             if (response.status === 401) {
               unref(ctrl).abort()
               return
+            } else if (response.status >= 500 || response.status === 404) {
+              throw new FatalError()
```

Comment:
It still retries 5 times for me 🤔  That is coming from our implementation I guess? You could set the `retryCounter` to the max limit before throwing the error.

And we should reduce the amounts of retries IMO, 3 should be sufficient.

Reply:
sure